src/components/radiobutton/RadioButton/radioButton.scss

```diff
@@ -0,0 +1,60 @@
+@import "../../../styles/theme";
+
+.radiobutton-wrapper {
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+
+  .text {
+    margin-left: $spacing_0_8;
+    font-size: $font_size_1_4;
+    font-weight: $font_medium;
+    color: &color_black_37;
+  }
+
+  & > input {
+    -webkit-appearance: none;
+    appearance: none;
+    background-color: $color_white_ff;
+    margin: 0;
+
+    width: 2rem;
+    height: 2rem;
+    border: $color_gray_92989b 0.2rem solid;
```

Comment:
Same here as in Checkbox component([comment link](https://github.com/UNICEFECAR/USupport-components-library/pull/13/files#r984075513))